### ISSUE TITLE
[cloud-compute-the-cosmos]:

### ISSUE THREAD

**Issue by zelszo** [googlecodelabs/feedback#477]
Hi Google Code Labs,

### **TL;DR**
A little update would be terrific for this Code Lab.

------------------------------------------------------------------------------------------------

I found this [lab](https://codelabs.developers.google.com/codelabs/cloud-compute-the-cosmos) very interesting and useful to improve tech skills. Taking the oportunity to get to know GCP products, but I found some "errors"/feedback:

1- On step **5. Creating a VM Instance with a custom VM Image** you are supposed to inser the code:

```
gcloud compute images create \
--source-uri \
http://storage.googleapis.com/codelab-2015-vm-image/1ad8c7f0540790f98eaf87801804feac985676e1.image.tar.gz \
codelab-image
```
But the current Cloud Shell will only work with:

```
gcloud compute images create codelab-image --source-uri=https://storage.googleapis.com/codelab-2015-vm-image/1ad8c7f0540790f98eaf87801804feac985676e1.image.tar.gz
```

2- Then on step **6. Creating a VM with an attached Persistent Disk** we have a similar situation where you are asked to insert:

```
gcloud compute instances create \
      codelab-node \
      --image codelab-image \
      --machine-type n1-standard-4 \
      --scopes compute-rw,storage-full \
      --boot-disk-device-name codelab-node \
      --disk name=disk1,device-name=disk1,mode=rw \
      --zone us-central1-f
```
I ended using: 

```
gcloud compute instances create codelab-node --image=codelab-image --machine-type=n1-standard-4 --scopes=compute-rw,storage-full --boot-disk-device-name=codelab-node --disk name=disk1,device-name=disk1,mode=rw --zone=us-central1-f 

```
I know the lab is not about copying/pasting but I got confused as soon as I started getting a lof of errors on the console.

3- On step **14. Assigning yourself part of the Universe** I was unable to dowload the .json response from the Tile Server App running on App Engine. 

4- Finally, I was confused on **16. Processing a tile** just because you are supossed to remove the <> from the actual command to insert on the console.

**Provided:**
```
gsutil cp <image-filename> gs://viewing-images-PROJECT_ID

```
**Actual Command**
```
gsutil cp image-filename.png gs://viewing-images-PROJECT_ID

```

Thanks,

**Comment by ZenMonk** [comment]
Thanks @zelszo!  I was struggling with this codelab, logged the issues, and then noticed your response.

**Comment by dangrgr** [comment]
Thanks @zelszo!
